src/services/neo4jService.js

```diff
@@ -53,7 +53,7 @@ export default {
     let res = [];
     const session = driver.session();
     await session
-      .run(`CALL db.labels() YIELD label RETURN label ORDER BY label`)
+      .run('CALL db.labels() YIELD label RETURN label ORDER BY label')
```

Comment:
this is not a template string so no need to use backticks. sorry, python linters spoiled me